### PR TITLE
fix default argusedtokens

### LINKED ISSUE #459: doubt about default value of avgUsedTokens
## Issue Description

the default traffic limiter has code
```
    private int avgUsedTokens(Node node) {
        if (node == null) {
            return -1;
        }
        return grade == RuleConstant.FLOW_GRADE_THREAD ? node.curThreadNum() : (int)node.passQps();
    }

```
I'm not sure why return -1 as default value.Seems Zero is more meaningful

Type: *bug report* or *feature request*

*bug report*

### Describe what happened (or what feature you want)

it return -1 as default value

### Describe what you expected to happen

expected return 0 as default value

### PR DESCRIPTION
**Pull request by mjaow**
### Describe what this PR does / why we need it

fix default value (-1 -> 0) of arg_used_tokens

### Does this pull request fix one issue?

Fixes #459 

### Describe how you did it

fix default value -1 to 0

### Describe how to verify it

CI

### CODE REVIEW

File: sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/RuleConstant.java [hunk 1]
```diff
@@ -53,6 +53,8 @@
 
     public static final int DEFAULT_SAMPLE_COUNT = 2;
     public static final int DEFAULT_WINDOW_INTERVAL_MS = 1000;
+    public static final int DEFAULT_AVG_USED_TOKENS = 0;
```
Comment (sczyh30) [inline sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/RuleConstant.java]: The `DEFAULT_AVG_USED_TOKENS` should not be put into `RuleConstant` as it is supposed to keep all constants for rules. You can put it inside the `DefaultController` directly and keep the original `RuleConstant`.

Comment (mjaow) [reply sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/RuleConstant.java]: reasonable

**Comment by mjaow** [comment]
fix is done

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**